_data/utilities.yml

```diff
@@ -179,6 +179,12 @@ websites:
       img: nicorgas.png
       tfa: No
 
+    - name: O2
+      url: http://www.o2.co.uk/
+      img: o2.png
+      tfa: Yes
+      sms: Yes
```

Comment:
Could you provide any documentation for how to set up two factor authentication?
If not, could you provide us with screenshots of the login process? (with personal information blurred out)

Reply:
Unfortunately, I don't have any such documentation on setting up 2fa. The login process is pretty simple, just username and password in a form. Where should I add such a screen shot?

Reply:
Unfortunately I don't have any documentation on how to set up 2fa. Where would you like me to put the screenshots of the login process?

Reply:
@rigoford you can just drag a photo to the comments field and it will automatically be added.
Like this:
![screenshot](https://user-images.githubusercontent.com/3535780/48240216-47150f80-e3d2-11e8-9fbc-582e692a94e0.gif)
😄

Reply:
@rigoford we're almost done here 😃